hs/src/Reach/Eval/Core.hs

```diff
@@ -5511,11 +5511,11 @@ doParallelReduce lhs (ParallelReduceRec {..}) = locAt slpr_at $ do
   let want lab = \case
         Just x -> return x
         Nothing -> expect_ $ Err_ParallelReduceIncomplete $ "missing " <> lab
-  inv_e <- want "invariant" pr_minv
```

Comment:
The docs say it is allowed multiple times (which I read as 1+) but I think you might have implemented many (0+)